types/event-map.d.ts

```diff
@@ -0,0 +1,72 @@
+declare type EventMapKey = string
+declare type LegacyKeyboardEventInit = {
+  charCode?: number
+} & KeyboardEventInit
+declare type EventMapValue =
+  | {
+      EventType: 'AnimationEvent'
```

Comment:
Do we want `EventType` here to be a string, or the type it represents from `lib.dom`?

```ts
EventType: 'AnimationEvent' | AnimationEvent
```

Reply:
The string is used at runtime:
https://github.com/testing-library/dom-testing-library/blob/281eb8b62700dec9a79ad315b6055a61f61f3c9f/src/events.js#L51-L58

So we need a TS macro to transform a TS interface name into string. I don't know how to do that.